linux/task.go

```diff
@@ -192,6 +192,16 @@ func (t *Task) DeleteProcess(ctx context.Context, pid uint32) (*plugin.Exit, err
 	}, nil
 }
 
+func (t *Task) Update(ctx context.Context, resources []byte) error {
+	_, err := t.shim.Update(ctx, &shim.UpdateTaskRequest{
+		Resources: &protobuf.Any{
+			TypeUrl: specs.Version,
```

Comment:
There is a current PR open for this but I don't see anything in the codebase right now